tfjs-backend-webgpu/README.md

```diff
@@ -1,13 +1,90 @@
-This is an experimental backend.
+# Usage
 
+This package adds a GPU accelerated [WebGPU](https://www.w3.org/TR/webgpu/) backend to TensorFlow.js. It currently supports
+the following models:
+- BlazeFace
+- BodyPix
+- Face landmarks detection
+- HandPose
+- MobileNet
+- PoseDetection
+- Universal sentence encoder
+- AutoML Image classification
+- AutoML Object detection
+- Speech commands
+
+Note that WebGPU 1.0 hasn't been officially released by W3C group. Currently WebGPU in chrome is in Origin Trial. To try webgpu features, you need to launch chrome canary browser with flag --enable-unsafe-webgpu.
```

Comment:
Not sure about the version schema of WebGPU, we can just remove "1.0" now.

Reply:
Suggest to change to this: Note that WebGPU hasn't been officially supported by browsers. Only Google Chrome is well tested, and its Canary Channel is highly recommended for your trial. Specify option "--enable-unsafe-webgpu" before you start the Chrome.